httpclient5-cache/src/main/java/org/apache/hc/client5/http/cache/HttpCacheStorageEntry.java

```diff
@@ -45,7 +45,7 @@
     private final HttpCacheEntry content;
 
     public HttpCacheStorageEntry(final String key, final HttpCacheEntry content) {
-        this.key = key;
+        this.key = Args.notNull(key, "Cache Key");
```

Comment:
Use the same name as the method arg;  `key`

Reply:
Changed. TY